sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs

```diff
@@ -1017,45 +1005,83 @@ public partial class AvailabilityProperties
+        public static Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState Building { get { throw null; } }
+        public static Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState Cancelled { get { throw null; } }
+        public static Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState Deleting { get { throw null; } }
+        public static Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState Failed { get { throw null; } }
+        public static Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState Pending { get { throw null; } }
+        public static Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState Succeeded { get { throw null; } }
+        public static Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState Updating { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState other) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override bool Equals(object obj) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.ResourceManager.Avs.Models.CloudLinkStatus left, Azure.ResourceManager.Avs.Models.CloudLinkStatus right) { throw null; }
-        public static implicit operator Azure.ResourceManager.Avs.Models.CloudLinkStatus (string value) { throw null; }
-        public static bool operator !=(Azure.ResourceManager.Avs.Models.CloudLinkStatus left, Azure.ResourceManager.Avs.Models.CloudLinkStatus right) { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState left, Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState left, Azure.ResourceManager.Avs.Models.AvsPrivateCloudProvisioningState right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class ClusterPatch
+    public partial class AvsSku
     {
-        public ClusterPatch() { }
-        public int? ClusterSize { get { throw null; } set { } }
-        public System.Collections.Generic.IList<string> Hosts { get { throw null; } }
+        public AvsSku(string name) { }
+        public string Name { get { throw null; } set { } }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public readonly partial struct ClusterProvisioningState : System.IEquatable<Azure.ResourceManager.Avs.Models.ClusterProvisioningState>
```

Comment:
I believe this should also have a prefix because its corresponding resource has been renamed

Reply:
Rename this type to `AvsPrivateCloudClusterProvisioningState`